js/app/weather.js

```diff
@@ -1,6 +1,10 @@
 function weatherForArea(area, target) {
-  var rate = _(DATA.WEATHER_RATES[area].weather_rates).find((r) => { return target < r[1]; });
-  return rate[0];
+  try {
```

Comment:
This should never fail if everything's set correctly.  I don't like that it will gobble up the error now.

Reply:
As far as i can remember fish located in Diadem can throw an error.

Reply:
This happend e.g. for the fish "Eurhinosaur" inside diadem (ff14angler id=445).

Reply:
Instead of catching exceptions, it should check if uhh... area is in DATA.WEATHER_RATES.
Exceptions are things you don't expect to occur; check for things you know you should check ;)